src/pages/home/report/ReportActionsView.js

```diff
@@ -127,7 +131,7 @@ class ReportActionsView extends React.Component {
 
         // If the reportID is not found then we have either not loaded this chat or the user is unable to access it.
         // We will attempt to fetch it and redirect if still not accessible.
-        if (!this.props.report.reportID) {
+        if (!this.props.report.reportID || !this.props.personalDetails[this.props.report.reportID]) {
```

Comment:
The `personalDetails` object has a key that is a `reportID`? IIRC `personalDetails` are keyed by logins.

Reply:
You are totally right about that... investigating more...

Reply:
Sorry for the long wait. I think I found a better way to handle this stuff:

Just like `Report.fetchChatReportsByIDs` chains a call to `PersonalDetails.getFromReportParticipants` [here](https://github.com/Expensify/App/blob/dd87f7e08c1af7c7fbb85a242d8b85221debc8c9/src/libs/actions/Report.js#L380), I chained a `PersonalDetails.getFromReportParticipants` call [here](https://github.com/Expensify/App/blob/dd87f7e08c1af7c7fbb85a242d8b85221debc8c9/src/libs/actions/Report.js#L756) when we call `Reports.fetchOrCreateChatReport`. This way we don't depend on `Report.fetchChatReportsByIDs` being called (which happens inconsistently depending on the platform as explained [here](https://github.com/Expensify/App/issues/9182#issuecomment-1139544946))

I know the new API refactor avoids chaining, but this is an easy fix, consistent with the old chaining approach and fixes the crash for IOS/Android, so I think it is fine as a temporary solution until all this gets refactored.